holoviews/util/__init__.py

```diff
@@ -339,6 +339,78 @@ def __call__(self, *args, **params):
         Store.current_backend = selected_backend
 
 
+def save(obj, filename, fmt='auto', backend=None, **kwargs):
+    """
+    Saves the supplied object to file.
+
+    The available output formats available depend on the backend being
+    used. By default and if the filename is a string the output format
+    will be inferred from the file extension. Otherwise an explicit
+    format will be specified. For ambiguous file extensions such as
+    html it may be necessary to specify an explicit fmt to override
+    the default.
+
+
+    Arguments
+    ---------
+    obj: HoloViews object
+        The HoloViews object to save to file
+    filename: string or IO object
+        The filename or BytesIO/StringIO object to save to
+    fmt: string
+        The format to save the object as, e.g. png, svg, html, or gif
+        and if widgets are desired either 'widgets' or 'scrubber'
+    backend: string
+        A valid HoloViews rendering backend, e.g. bokeh or matplotlib
+    **kwargs: dict
+        Additional keyword arguments passed to the renderer,
+        e.g. fps for animations
+    """
+    backend = backend or Store.current_backend
+    renderer_obj = renderer(backend)
+    if kwargs:
+        renderer_obj = renderer_obj.instance(**kwargs)
+    if isinstance(filename, basestring):
+        supported = [mfmt for tformats in renderer_obj.mode_formats.values()
+                     for mformats in tformats.values() for mfmt in mformats]
+        formats = filename.split('.')
+        if fmt == 'auto' and formats and formats[-1] != 'html':
+            fmt = formats[-1]
+        if formats[-1] in supported:
+            filename = '.'.join(formats[:-1])
+    return renderer_obj.save(obj, filename, fmt=fmt)
+
+
+def render(obj, backend=None, **kwargs):
+    """
+    Renders the object to a figure object.
```

Comment:
Renders the HoloViews object to the corresponding object in the specified backend, e.g. a Matplotlib or Bokeh figure.

The backend defaults to the currently declared default backend. The resulting object can then be used with other objects in the specified backend.  For instance, if you want to make a multi-part Bokeh figure using a plot type only available in HoloViews, you can use this function to return a Bokeh figure that you can use like any hand-constructed Bokeh figure in a Bokeh layout.

Reply:
Sounds good.